pkg/registry/service/portallocator/allocator.go

```diff
@@ -151,8 +151,7 @@ func (r *PortAllocator) Restore(pr util.PortRange, data []byte) error {
 	if !ok {
 		return fmt.Errorf("not a snapshottable allocator")
 	}
-	snapshottable.Restore(pr.String(), data)
-	return nil
+	return snapshottable.Restore(pr.String(), data)
```

Comment:
Oh wow - I think you caught this one before as well, but I must have lost it during the git operations.  I will do a diff to see what else I may have lost....